cedar-policy-core/src/authorizer.rs

```diff
@@ -40,6 +40,7 @@ pub use partial_response::ErrorState;
 pub use partial_response::PartialResponse;
 
 /// Authorizer
+#[derive(Clone)]
```

Comment:
May as well derive `Debug` while we're in here

```suggestion
#[derive(Debug, Clone)]
```